docs/index.md

```diff
@@ -339,6 +339,14 @@ have a `kid` header with a defined key.
 
 Default is `False`.
 
+### JWT_REQUIRE_TOKEN_ID
+
+Expect all tokens to have a `jti` token id claim, and refreshable tokens to have an `orig_jti` claim.
+
+These claims have been added automatically since version ???. Once all valid tokens in your environment have these claims, we recommend setting this to `True`.
```

Comment:
Could you please add an example when it would be appropriate to turn this setting on? It may be ambiguous to some users that this implicitly depends on the lifetime of their tokens in production and that it's not safe to turn it on whenever.

Reply:
This is a bit too dense. I'll try and explain it more clearly.

Reply:
Also, for the version number, do you think this will go out as 1.17, or a new patch level for 1.16?